### ISSUE TITLE
Need Permission/Guidence

### ISSUE THREAD

**Issue by ved009** [markuta/go-security-txt#5]
Hello,

I am not sure if this issue will be looked upon, but wanted to publish this data on a website for security researchers to find security based programs. 

I just wanted to check if there will be any copyright infringement/law suit in case I use this program and my own collected list of root domains and then publish it on my website

**Comment by markuta** [comment]
Hey @ved009, that should be fine just link the source on your website.

**Comment by ved009** [comment]
Definitely, will do! I will also share it here for you to take a look. 

Thanks for the confirmation
